components/chassis.py

```diff
@@ -104,7 +111,9 @@ def set(self, desired_state: SwerveModuleState):
         )
 
     def zero(self):
```

Comment:
Should this still be called `zero`, seeing as it doesn't actually zero? Should it be more like "synchronise motor sensor to encoder"?